packages/node_modules/@webex/plugin-meetings/src/roap/util.js

```diff
@@ -45,54 +43,16 @@ RoapUtil.ensureMeeting = (meeting, type) => {
   return true;
 };
 
-RoapUtil.compareLastRemoteOffer = (currentOffer, previousOffer) => {
-  let index1, index2;
-  let difference = false;
+RoapUtil.updatePeerConnection = (meeting, session) => PeerConnectionManager.updatePeerConnection({
+  offerSdp: session.OFFER.sdps,
+  peerConnection: meeting.mediaProperties.peerConnection
+},
+meeting.id)
+  .then((res) => {
+    meeting.roap.lastRoapOffer = session.OFFER.sdps;
 
-  if (isEmpty(previousOffer)) {
-    return true;
-  }
-
-  // TODO: fix roap issue where chrome cannot create offer after replaceTrack and createOffer
-  if (previousOffer.length > 0 && previousOffer.length === currentOffer.length) {
```

Comment:
we can't just compare the S_LINE, we need to look at all of it. and it will always be different.

Reply:
Do we know what this method was trying to accomplish before? What's the effect of no longer doing comparisons?